Makefile

```diff
@@ -136,7 +136,13 @@ up: build  ## start develop/test environment
 ifneq ($(NETWORK_MODE),host)
 	docker network inspect $(NETWORK_MODE) &> /dev/null || ([ $$? -ne 0 ] && docker network create $(NETWORK_MODE))
 endif
-	$(DOCKER_COMPOSE_ARGS) docker-compose up -d
+	$(DOCKER_COMPOSE_ARGS) docker-compose \
+	-f docker-compose/tango-docker-compose.yml \
+	-f docker-compose/tmc-docker-compose.yml \
```

Comment:
I would suggest starting the CSP and SDP devices first before starting TMC devices. This will help to avoid failed subscription at TMC if CSP/SDP devices are not ready. Change only if required.

Reply:
Done.

Reply:
Commit ID : [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]